RecoEgamma/EgammaElectronAlgos/src/GsfElectronAlgo.cc

```diff
@@ -500,10 +504,10 @@ GsfElectronAlgo::EventData GsfElectronAlgo::beginEvent(edm::Event const& event,
           egHcalIsoConeSizeOutSmall,
           EgammaHcalIsolation::InclusionRule::withinConeAroundCluster,
           egHcalIsoConeSizeIn,
-          hcalHelperCone_.eThresHB(),
+          EgammaHcalIsolation::arrayHB{{0., 0., 0., 0.}},
```

Comment:
@mandrenguyen I think this and the following similar changes are the reason why we still see differences. I just meant to keep these lines untouched (they are fixed in #43355), but in the meantime this PR was basically rebased, so by doing this I am modifying them again.